packages/sanes-chrome-extension/public/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "manifest_version": 2,
   "name": "IOV - Chrome extension",
-  "version": "0.0.0",
+  "version": "will_be_set_in_build_step",
```

Comment:
This is because for updating the extension, we need to increase version, right? Which is the version nomenclature?

Reply:
The version is copied 1:1 from from package.json into manifest.json during the build. This is just a placeholder to make clear that the manifest version is auto-generated.

Reply:
👌